src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -9,6 +9,27 @@
 
 namespace BugsnagUnity
 {
+    internal class DisposableContainer
+    {
+        private List<IDisposable> _disposables = new List<IDisposable>();
+
+        public void Add(IDisposable disposable)
+        {
+            _disposables.Add(disposable);
+        }
```

Comment:
Not sure if this is idiomatic, but by adding a little generic this method could return whatever was passed in which would simplify its use:

```suggestion
        public V Add<V>(V disposable) where V : IDisposable
        {
            _disposables.Add(disposable);
            return disposable;
        }
```

which would allow a more concise:
```
var arrayList = disposableContainer.Add(new AndroidJavaObject("java.util.ArrayList"));
```

Reply:
I would prefer to keep it as is for readability